### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by PlanetThoughts** [cyanfish/naps2#163] [General]
Thank you for updating this very helpful piece of software. I made a contribution. The biggest and only problem I had with the 2019 version was that if paper got jammed while scanning, there was no way to cancel the operation and the scanner got locked as "in use." That required rebooting the computer to free up the device. Has that been changed / fixed?

**Comment by cyanfish** [comment]
You can try with the latest [beta version](https://www.naps2.com/download). It may or may not make a difference. You can also try switching between WIA and TWAIN drivers in your profile settings as one may work better than the other.

**Comment by cvraosan** [comment]
hi Ben
tried to pay you 10USD using Paypal -- it says you dont accept donations from India 
HAHA
find a solution to this bug

**Reply by cyanfish** [reply]
Appreciate it, you might be able to use https://paypal.me/naps2 if you want to try.

**Comment by cvraosan** [comment]
tried it and it also refused

￼

**Comment by cvraosan** [comment]
tried paypal and it refused
[PastedGraphic-1.png.pdf](https://github.com/cyanfish/naps2/files/13696044/PastedGraphic-1.png.pdf)
